api/migrations/20201123161901_modify-users.js

```diff
@@ -20,6 +20,7 @@ exports.up = async knex => {
     table
       .string('uid')
       .notNullable()
+      .defaultTo('not set')
```

Comment:
I don't think this will work, since there's a unique constraint set on line 25.

Reply:
should we clear out the table then